http/mdm/mdm.go

```diff
@@ -61,12 +70,20 @@ func CommandAndReportResultsHandler(svc service.CommandAndReportResults, logger
 		}
 		respBytes, err := service.CommandAndReportResultsRequest(svc, mdmReqFromHTTPReq(r), bodyBytes)
 		if err != nil {
-			logger.Info("msg", "command report results", "err", err)
+			logs := []interface{}{"msg", "command report results"}
 			httpStatus := http.StatusInternalServerError
 			var statusErr *service.HTTPStatusError
 			if errors.As(err, &statusErr) {
 				httpStatus = statusErr.Status
+				err = statusErr.Unwrap()
+			}
+			var parseErr *mdm.ParseError
+			if errors.As(err, &parseErr) {
+				logs = append(logs, "content", string(parseErr.Content))
+				err = parseErr.Unwrap()
```

Comment:
just a lil bit more context :)

```suggestion
				err = fmt.Errorf("parse error: %w", parseErr.Unwrap())
```